pkg/hubble/observer/observeroption/option.go

```diff
@@ -44,10 +45,8 @@ type Server interface {
 
 // Options stores all the configurations values for the hubble server.
 type Options struct {
-	// Both sizes should really be uint32 but it's better saved for a single
-	// refactor commit.
-	MaxFlows      int // max number of flows that can be stored in the ring buffer
-	MonitorBuffer int // buffer size for monitor payload
+	MaxFlows      container.Capacity // max number of flows that can be stored in the ring buffer
+	MonitorBuffer int                // buffer size for monitor payload FIXME: change for uint32
```

Comment:
Should this FIXME be addressed or deleted?

Reply:
I think it should be addressed (cc @gandro) but it's completely unrelated to this PR. The FIXME was referring to both `MaxFlows` and `MonitorBuffer` but with this PR it becomes only valid for `MonitorBuffer`.

Reply:
It's fully unrelated to the PR. We can leave it there, though I'm not sure actually making `MonitorBuffer` a uint32 is idiomatic Golang (it's used for the channel capacitiy, which in Golang seems to be always an int, and not a uint32)

Reply:
Somehow I was convinced you were the author of this comment which is why I cc'd you but it's seems the author is @glibsm in https://github.com/cilium/hubble/pull/183. I'm unsure about the rationale but I agree with you that `int` is idiomatic Go.

Reply:
While most of these should technically be unsigned, `int` offers a nice `-1` option for special behavior. 

I'd also argue spending one bit on sigh when these clearly don't have any issues to use `int` is not a big deal

Reply:
Based on this discussion, I removed the `FIXME` note.